packages/main/src/plugin/docker/docker-context-handler.ts

```diff
@@ -36,19 +36,43 @@ export interface DockerContextParsingInfo extends Omit<DockerContextInfo, 'isCur
  * Handle the `docker context`, allowing to list them and switch between them.
  */
 export class DockerContextHandler {
-  protected getDockerConfigPath(): string {
-    return join(homedir(), '.docker', 'config.json');
+  protected getDockerConfig(): string {
+    // DOCKER_CONFIG is defines path to docker client configuration files
+    // Docker uses ~/.docker if DOCKER_CONFIG is not set
+    // see: https://docs.docker.com/reference/cli/docker/ for more details
+    const dockerConfig = process.env['DOCKER_CONFIG'];
```

Comment:
the problem is that we can't read the env variable.

on macOS, the applications are started without having access to the env variables (that you will define in your login script)

Reply:
right, it completely slipped my mind :c

and i guess i understood, why it works in test suite - if i run ui via `pnpm watch` to test, it inherits all env from my shell

sooo, i think we have two options here: either call `docker` via `processApi.exec()` or add another option into settings with `DOCKER_CONFIG`, `DOCKER_HOST` and `DOCKER_CONTEXT`

Reply:
i mean, if second: besides reading envs, fetch values from settings (settings would have priority over envs)

Reply:
...but first is easier to implement

Reply:
I guess that if you call docker cli it will not have the environment variable as well so the CLI won't find it neither

Reply:
yeah, it slipped either

so, i'll make a few options in Settings->Preferences->Docker Compatibility, right after "Enabled" checkbox

good?

Reply:
I think we will need to move the logic of this part out of rodman desktop core and put it in the docker extension and here we will be able to put the preferences

Need to think on how it can be integrated well through the new docker compatibility section

Reply:
@benoitf may we use same workaround as [here](https://github.com/podman-desktop/podman-desktop/pull/10086) to make this work?

Reply:
yes but the issue is that it needs to be part of the docker extension, not of Podman Desktop core